keyboards/keebwerk/nano_slider/config.h

```diff
@@ -0,0 +1,223 @@
+/*
+Copyright 2020 Duckle
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x03A8
+#define PRODUCT_ID 0x0000
+#define DEVICE_VER 0x0001
+#define MANUFACTURER keebwerk.
+#define PRODUCT nano. slider
+#define DESCRIPTION A small eight key macropad
+
+/* key matrix size */
+#define MATRIX_ROWS 2
+#define MATRIX_COLS 4
+
+/*
+ * Keyboard Matrix Assignments
+ */
+#define MATRIX_ROW_PINS \
+    { F0, F1 }
+#define MATRIX_COL_PINS \
+    { B0, B1, B2, B3 }
+#define UNUSED_PINS
+
+#define SLIDER_PIN D4
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+// #define BACKLIGHT_PIN B7
```

Comment:
According to [this](https://www.keebwerk.com/wp-content/uploads/2019/09/IMG_3659-1067x800.jpg), there is in-switch backlight support. @RaySoyama any idea what pin it's on?

Reply:
How would I check this? I have the RGB working on my slider, would I check on my PCB? @fauxpark

Reply:
One of the pins on the MCU will be connected to one of the pins on Q1 (possibly through R19; I can't quite make out the value). If you have a multimeter handy, a simple continuity check should do.

Reply:
I talked to keeb.werk, and it is indeed on B7 :)